ga4gh/datamodel/reads.py

```diff
@@ -68,6 +68,8 @@ class SamFlags(object):
     """
     NUMBER_READS = 0x1
     PROPER_PLACEMENT = 0x2
+    UNMAPPED = 0x4
+    NEXT_SEGMENT_UNMAPPED = 0x8
```

Comment:
0x4 and 0x8 both set: this is equivalent to `readNumber` being a value other than 1 or 0 - the ga4gh spec is just being more specific about which index it is. The spec is more expressive/specific than the BAM spec here, so it may be difficult to do a direct conversion from a SAMRecord without actually traversing mates.

0x4 and 0x8 both unset: I would interpret this as translating to a `null` `readNumber`, though we should include this in the documentation.